tests/unit/index-nodetest.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+/* globals require, describe, before, beforeEach, it, process */
```

Comment:
Sorry about the incidental change here. Adding this calmed down my editor's jshinting. I'd be happy to remove it if undesired.